doc/sphinx/language/gallina-extensions.rst

```diff
@@ -951,83 +944,39 @@ Reserved commands inside an interactive module
     .. exn:: This is not the last opened module.
        :undocumented:
 
-.. cmd:: Module @ident := @module_expression
-
-    This command defines the module identifier :token:`ident` to be equal
-    to :token:`module_expression`.
-
-    .. cmdv:: Module @ident {* @module_binding} := @module_expression
-
-       Defines a functor with parameters given by the list of :token:`module_binding` and body :token:`module_expression`.
-
-    .. cmdv:: Module @ident {* @module_binding} : @module_type := @module_expression
-
-       Defines a functor with parameters given by the list of :token:`module_binding` (possibly none), and output module type :token:`module_type`,
-       with body :token:`module_expression`.
-
-    .. cmdv:: Module @ident {* @module_binding} <: {+<: @module_type} := @module_expression
-
-       Defines a functor with parameters given by module_bindings (possibly none) with body :token:`module_expression`.
-       The body is checked against each :n:`@module_type__i`.
-
-    .. cmdv:: Module @ident {* @module_binding} := {+<+ @module_expression}
-
-       is equivalent to an interactive module where each :token:`module_expression` is included.
```

Comment:
This was forgotten in the description of `Module`.

Reply:
While there is a direct relationship, how can I clearly and simply state how they're equivalent (e.g. at a syntactic level).  Interactively the `module_expression`s would appear in different commands--what are those commands?

Not sure this is really that important a point to make.

Reply:
>  Interactively the `module_expression`s would appear in different commands--what are those commands?

`Include`

> Not sure this is really that important a point to make.

I'm just ensuring we are not losing documentation while making these changes.